java/third-party/mockito/LICENSE

```diff
@@ -0,0 +1,21 @@
+The MIT License
```

Comment:
Since we're now fetching these at build time, rather than bundling the jars, we don't need to include the various licenses for these third-party components; can you remove those?